java/ql/src/semmle/code/java/frameworks/spring/SpringStringUtils.qll

```diff
@@ -0,0 +1,58 @@
+import java
+private import semmle.code.java.dataflow.ExternalFlow
+
+private class SpringStringUtilsModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row = [
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];Argument[0];taint",
+        "org.springframework.util;StringUtils;false;applyRelativePath;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;applyRelativePath;;;Argument[1];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;arrayToCommaDelimitedString;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;arrayToDelimitedString;;;Argument[0];ReturnValue;taint", // for review: this is "org.springframework.util;StringUtils;false;deliminate array with string argument[1]"; should I consider the delimiter to taint the result as well?
```

Comment:
It goes in there, so yes (imagine if some careless person allowed a remote user to say whether the sep should be `,` or `;` or `;"; DROP TABLE universe`). Same for other functions inserting a delimiter unless the delimiter is a char